clean_old_aggregates.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import pytz
+import sys
+import argparse
+import os
+import itertools
+from hammers import osapi, osrest
+from django.utils import timezone
+from hammers.osrest.nova import _addremove_host
+
+# Set env variables
+os.environ["OS_AUTH_URL"] = "http://10.20.111.241:35357/v3"
```

Comment:
Hammers will run with these environment variables already set. In general we shouldn't be setting environment variables; they should be set externally to the script.

FYI using env as configuration is part of the [12-factor app model](https://12factor.net/), which I find to be a good way to write things.